t/stg

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
+
+# This script is only used by the tests so they can work without stgit being
+# installed. It assumes PYTHONPATH to be set up correctly.
```

Comment:
I think we want to avoid making changes like this to the test infra. The test infra is heavily borrowed from git -- every change we make makes it more difficult to update our infra to match git's. If think is something we think is important the best place to add it might be the `git` repo itself.

Is this something that's needed for Windows?

Reply:
Sorry for being MIA for so long... Yes, I agree, it is bad. Maybe move it out of `t/`? As to your question: no, it is more related to setuptool packaging.